src/test/java/org/folio/rest/api/StorageTestSuite.java

```diff
@@ -128,44 +129,7 @@ public static void before()
 
     vertx = Vertx.vertx();
 
-    String useExternalDatabase = System.getProperty(
-      "org.folio.circulation.storage.test.database",
-      "embedded");
-
-    switch (useExternalDatabase) {
-      case "environment":
-        log.info("Using environment settings");
-        break;
-
-      case "external":
-        String postgresConfigPath = System.getProperty(
-          "org.folio.circulation.storage.test.config",
-          "/postgres-conf-local.json");
-
-        log.info(String.format(
-          "Using external configuration settings: '%s'", postgresConfigPath));
-
-        PostgresClient.setConfigFilePath(postgresConfigPath);
-        break;
-
-      case "embedded":
-        log.info("Using embedded PostgreSQL");
-
-        PostgresClient.setIsEmbedded(true);
-        PostgresClient.setEmbeddedPort(NetworkUtils.nextFreePort());
-
-        PostgresClient client = PostgresClient.getInstance(vertx);
-        client.startEmbeddedPostgres();
-        break;
-
-      default:
-        String message = "No understood database choice made."
-          + "Please set org.folio.circulation.storage.test.config"
-          + "to 'external', 'environment' or 'embedded'";
-
-        log.error(message);
-        assert false;
-    }
+    PostgresClient.setPostgresTester(new PostgresTesterContainer());
```

Comment:
This is much nicer than what I used to need to do, thank you.

Reply:
Thx